### ISSUE TITLE
PowerSuits breaks other armor textures

### ISSUE THREAD

**Issue by IamRyan30** [lehjr/MachineMusePowersuits#53]
**Version info**
Java version: 1.8.0_51
Minecraft Forge version: 36.2.21
MPS version: ModularPowersuits-1.16.5-2.1.22.jar (older versions don't work either)
Numina Version: Numina-1.16.5-2.1.22.jar (newer versions don't work either)
**I downgraded to this versions of the mod because in the newer ones, the Power Fist modules, don't work.**

**Describe the bug**
With Modular Powersuit, all the armors in the game, look bad. I have no idea what mod breaks Modular Powersuit, but I wanted to report this bug anyway.

**Mod list**
[Here.](https://gist.github.com/IamRyan30/c16affc694bcfc6ecc9b4df3a90fa224)

**Screenshots**
![image](https://user-images.githubusercontent.com/91336389/164886183-c8e1e675-bcba-4a3b-8100-445b1b81bb2b.png)
![image](https://user-images.githubusercontent.com/91336389/164886192-4ae51fbe-1cab-48a8-9d68-4a245314840b.png)

**Comment by lehjr** [comment]
I'll be marking this as INVALID, since it's the latest version of the mod, and not the smallest mod list needed to reproduce the issue. I simply don't have the kind of time it requires to hunt down your mods, download and install them, then eliminated them until i can no longer replicate the issue. It simply isn't feasible. I'd be spending my days doing that and the mod would be stuck in limbo.

**Comment by IamRyan30** [comment]
> I'll be marking this as INVALID, since it's NOT the latest version of the mod, and not the smallest mod list needed to reproduce the issue. I simply don't have the kind of time it requires to hunt down your mods, download and install them, then eliminated them until i can no longer replicate the issue. It simply isn't feasible. I'd be spending my days doing that and the mod would be stuck in limbo.

1. I clearly said that this also happens on the newer versions - I've tested all of them, including - ModularPowersuits-1.16.5-2.2.24.jar.
2. I also clearly said that this issue happens just with your mod. That mod list was provided just so you know where the issue was initially discovered. Fine, I'll remove the mod list.
3. Want to recreate this? Install just MPS with Atum2 and there you go. You can replace Atum2 with Botania, or any other mod which adds custom armor models and you'll have the same issue.

**Comment by lehjr** [comment]
You gave me a list of over 200 mods, and said, here's my list. "UPDATE: Yes. Bug from Modular Powersuit entirely. The Vanilla Armors look normal, but the ones from other mods, don't." is not the same as mods with custom armor models.

**Comment by lehjr** [comment]
and yes, I can see your edits.

**Comment by IamRyan30** [comment]
> You gave me a list of over 200 mods, and said, here's my list. "UPDATE: Yes. Bug from Modular Powersuit entirely. The Vanilla Armors look normal, but the ones from other mods, don't." is not the same as mods with custom armor models.

Yes, and I said that I gave you that list so you know where the issue was initially discovered. And I think it's pretty obvious that I was talking about armors with custom models, since that's the type of armors shown in the pictures that I left here.

**Comment by lehjr** [comment]
just because a mod has custom armor doesn't mean it has a custom armor model.

**Comment by IamRyan30** [comment]
> just because a mod has custom armor doesn't mean it has a custom armor model.

Yeah, apologies on that, apparently it was a misconception on my part. So, custom armors with vanilla models have no problem, but custom armors with custom models, do.

**Comment by IamRyan30** [comment]
> Confirmed and fixed the issue. I'll have to track down the power fist issue before the next release though.

Don't know if this helps, but I just discovered that the power fist issue takes place at random. I lowered the amount of mods present on my list, and I though I was getting somewhere, but to no success. Sometimes the textures are fine, and sometimes, they're missing. I could keep removing the mods until I don't see the missing textures for multiple times when I open the game, but I doubt how accurate/how useful would that be overall.

**Comment by lehjr** [comment]
Might be a loading order issue compounded by nearly everything being multithreaded. I’ll have to look into it a bit more.
